src/main/java/com/gg/server/admin/tournament/service/TournamentAdminService.java

```diff
@@ -0,0 +1,89 @@
+package com.gg.server.admin.tournament.service;
+
+import com.gg.server.admin.tournament.dto.TournamentAdminUpdateRequestDto;
+import com.gg.server.domain.tournament.data.Tournament;
+import com.gg.server.domain.tournament.data.TournamentRepository;
+import com.gg.server.domain.tournament.exception.TournamentConflictException;
+import com.gg.server.domain.tournament.exception.TournamentNotFoundException;
+import com.gg.server.domain.tournament.exception.TournamentUpdateException;
+import com.gg.server.domain.tournament.type.TournamentStatus;
+import com.gg.server.global.exception.ErrorCode;
+import com.gg.server.global.exception.custom.InvalidParameterException;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class TournamentAdminService {
+    private final TournamentRepository tournamentRepository;
+    public static final long ALLOWED_MINIMAL_START_TIME = 2;
+    public static final long MINIMUM_TOURNAMENT_DURATION = 2;
```

Comment:
```suggestion
    // 토너먼트 최소 시작 날짜 (n일 후)
    private static final int ALLOWED_MINIMAL_START_TIME = 2;
    // 토너먼트 최소 진행 시간 (n시간)
    private static final int MINIMUM_TOURNAMENT_DURATION = 2;
```
외부에서 사용하지 않으면 일단 private으로 만들어두는 게 좋을 것 같아요
그리고 자료형도 int로 충분할 것 같슴다! 
상수명으로 이해하기 어려울 것 같아서 주석도 있으면 좋을 것 같아용

Reply:
넵 수정하겠습니다!